### PR TITLE
Trendlines scheduled views

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: view-list-scheduled-views
 title: View Information About Scheduled Views
 description: Learn how to view a list of scheduled views configured for your org, and to view the details of a scheduled view.
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:4]:
```suggestion
description: Learn how to view a list of scheduled views configured for your org and to view the details of a scheduled view.
```

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 2]
```diff
@@ -19,13 +19,14 @@ You must have a role that grants you the View Scheduled Views [role capability
     * **Data Forwarding**. If the scheduled view is configured to forward data to an S3 bucket, this column contains the name of the [data forwarding](../data-forwarding/amazon-s3-bucket.md) destination.
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:19]: This doesn't read well. It's not clear which specific column is being referred to.

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 3]
```diff
@@ -19,13 +19,14 @@ You must have a role that grants you the View Scheduled Views [role capability
     * **Data Forwarding**. If the scheduled view is configured to forward data to an S3 bucket, this column contains the name of the [data forwarding](../data-forwarding/amazon-s3-bucket.md) destination.  
 1. To view details of a scheduled view configuration, click the row that contains the view. <br/>![sched-view-more-options.png](/img/scheduled-views/sched-view-more-options.png)
 1. A pane pops up on the right side of the page with the following information:
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:21]:
```suggestion
1. A pane pops up on the right side of the page with the following information.
```

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 4]
```diff
@@ -19,13 +19,14 @@ You must have a role that grants you the View Scheduled Views [role capability
...
-   * Scheduled View Name
-   * Query. The query that returns that data to be written to the scheduled view
-   * Search Mode.
-   * Progress. Indicates how up-to-date the scheduled view is.
-   * Retention Period. The period of time data in the scheduled view is retained.
-   * Start Date. The date that data was first added to the scheduled view.
-   * Lag Time. If the scheduled view is not up-to-date, **Lag Time** contains the actual lag time. For more information, see [Scheduled View Lag Time](scheduled-view-lag-time.md)
+    * **Name**. Displays the name of the scheduled view.
+    * **Query**. The query that returns that data to be written to the scheduled view
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:23]:
```suggestion
    * **Query**. The query that returns the data for the scheduled view.
```

Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:23]:
```suggestion
    * **Query**. The query that returns the data for the scheduled view.
```

**Review by JainM6 — DISMISSED** [review]
Formatting changes

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

File: docs/manage/scheduled-views/view-list-scheduled-views.md (hunk 2 above)
Comment (JV0812) [reply docs/manage/scheduled-views/view-list-scheduled-views.md]: updated.

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 5]
```diff
@@ -16,16 +16,17 @@ You must have a role that grants you the View Scheduled Views [role capability
     * **Name**. The name assigned to the scheduled view.
     * **Storage Consumed**. The total volume of uncompressed data ingested across the duration of the retention period.
     * **Retention Period**. The number of days configured as the retention period.
-    * **Data Forwarding**. If the scheduled view is configured to forward data to an S3 bucket, this column contains the name of the [data forwarding](../data-forwarding/amazon-s3-bucket.md) destination.  
+    * **Data Forwarding**. Indicates the name of the [data forwarding](../data-forwarding/amazon-s3-bucket.md) destination, if the scheduled view is configured to forward data to the S3 bucket.
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:19]:
```suggestion
    * **Data Forwarding**. Indicates the name of the [data forwarding](../data-forwarding/amazon-s3-bucket.md) destination if the scheduled view is configured to forward data to the S3 bucket.
```

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 6]
```diff
@@ -16,16 +16,17 @@ You must have a role that grants you the View Scheduled Views [role capability
...
-   * Progress. Indicates how up-to-date the scheduled view is.
-   * Retention Period. The period of time data in the scheduled view is retained.
-   * Start Date. The date that data was first added to the scheduled view.
-   * Lag Time. If the scheduled view is not up-to-date, **Lag Time** contains the actual lag time. For more information, see [Scheduled View Lag Time](scheduled-view-lag-time.md)
+1. A pane pops up on the right side of the page with the following information.
+    * **Name**. Displays the name of the scheduled view.
+    * **Query**. The query that returns the data for the scheduled view.
+    * **Search Mode**. Indicates the type of search mode, such as Manual Mode or Auto Parse Mode.
+    * **Progress**. Indicates how up-to-date the scheduled view is.
+    * **Data scanned to fill this scheduled view**.Provides trend information about the data scanned over time and displays the total data scanned for the selected time to run the query.
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:26]:
```suggestion
    * **Data scanned to fill this scheduled view**. Provides trend information about the data scanned over time and displays the total data scanned for the selected time to run the query.
```

File: docs/manage/scheduled-views/view-list-scheduled-views.md [hunk 7]
```diff
@@ -16,16 +16,17 @@ You must have a role that grants you the View Scheduled Views [role capability
...
+    * **Retention Period**. The period of time data in the scheduled view is retained.
+    * **Start Date**. The date that data was first added to the scheduled view.
```
Comment (JainM6) [inline docs/manage/scheduled-views/view-list-scheduled-views.md:28]:
```suggestion
    * **Start Date**. Date when data was first added to the scheduled view.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting changes

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting changes

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting changes

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting changes

**Review by kimsauce — APPROVED** [review]

**Pull request merged**